src/GraphQl/Resolver/Factory/ItemMutationResolverFactory.php

```diff
@@ -89,7 +89,17 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
             switch ($operationName) {
                 case 'create':
                 case 'update':
-                    unset($args['input']['id']);
+                    // Invert _id with id because GraphQL is using a different convention than API Platform.
```

Comment:
For the record, I think we should use this convention (id = IRI, _id = the field) everywhere when a specific id isn't defined by the implemented spec (everywhere except in JSON-LD currently).

Reply:
Okay for `id` (it's in the generic item normalizer), but where it is in the code for `_id`? I don't think the AbstractItemNormalizer can understand it.